src/main/java/com/facebook/openwifirrm/ucentral/informationelement/BssAcAccessDelay.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.informationelement;
+
+import java.util.Objects;
+
+import com.google.gson.JsonObject;
+
+/**
+ * This information element (IE) appears in wifiscan entries. It currently does
+ * not appear in these entries AFAICT. It's called "BSS AC Access Delay" in
+ * 802.11 specs (section 9.4.2.43). Refer to the specification for more details.
+ * Language in javadocs is taken from the specification.
+ */
+public class BssAcAccessDelay {
+	/** Defined in 802.11 table 9-92 */
+	public static final int TYPE = 68;
+
+	/**
+	 * Subfield that goes into Access Category Access Delay field in BSS AC
+	 * Access Delay. For information on what the values mean, check section
+	 * 9.4.2.43
+	 */
+	public static class AccessCategoryAccessDelay {
+		/**
+		 * Unsigned 8 bits that represents a scaled representation of best effort AC
+		 * access delay
+		 */
+		public final short averageAccessDelayForBestEffort;
+		/**
+		 * Unsigned 8 bits that represents a scaled representation of background AC
+		 * access delay
+		 */
+		public final short averageAccessDelayForBackground;
+		/**
+		 * Unsigned 8 bits that represents a scaled representation of video AC access
+		 * delay
+		 */
+		public final short averageAccessDelayForVideo;
+		/**
+		 * Unsigned 8 bits that represents a scaled representation of voice AC access
+		 * delay
+		 */
+		public final short averageAccessDelayForVoice;
+
+		/** Constructor */
+		public AccessCategoryAccessDelay(
+			short averageAccessDelayForBestEffort,
+			short averageAccessDelayForBackground,
+			short averageAccessDelayForVideo,
+			short averageAccessDelayForVoice
+		) {
+			this.averageAccessDelayForBestEffort =
+				averageAccessDelayForBestEffort;
+			this.averageAccessDelayForBackground =
+				averageAccessDelayForBackground;
+			this.averageAccessDelayForVideo = averageAccessDelayForVideo;
+			this.averageAccessDelayForVoice = averageAccessDelayForVoice;
+		}
+
+		/** Parse AccessCategoryAccessDelay from JSON object */
+		// TODO rename fields as necessary - we don't know how the data format yet
```

Comment:
I don't understand this TODO. The field names seem perfect, they are just the 802.11 field names in camel case, right? And if the data format is different for some reason from the 802.11 definition, why would that make us change the name of the variable?

Same for other TODOs like this.